File: Model/Employee.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SOLID_example_2.Model
{
    public class Employee
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public bool Status { get; set; }
    }
}

File: UI/EmployeeSolidCRUD/src/app/data/type/employee.ts
export class Employee{
    constructor(public id: number, 
                public name: string, 
                public phoneNumber: string, 
                public email: string,
                public status: boolean){
        this.id= id;
        this.name= name;
        this.phoneNumber= phoneNumber;
        this.email= email;
        this.status= status;
    }
}